api/interestr/api/serializers.py

```diff
@@ -3,6 +3,8 @@
 from . import models as core_models
 from django.contrib.auth import models as auth_models
 
+from serializer_utils import get_or_none
+
```

Comment:
Python could not find the module. Since "get_or_none" is never used, I think we can safely delete this line.

Reply:
Interesting, I thought I had deleted this... lol